gdal_subsetter/transform.py

```diff
@@ -1405,29 +1410,26 @@ def _process_geogcs(ds_in, dst):
                 if gt[2] == 0.0 and gt[4] == 0.0:
                     datavar.coordinates = 'lon lat'
 
-        # open infile
-        ds_in = gdal.Open(infile)
-        # create a output nc file
-        dst = Dataset(outfile, mode='w', format='NETCDF4')
-        # define global attributes
-        dst.title = ''
-        dst.institution = 'Alaska Satellite Facility'
-        dst.source = ''
-        dst.references = ''
-        dst.comment = ''
-        dst.history = f'{datetime.utcnow():%d/%m/%Y %H:%M:%S} (UTC)'
-        dst.GDAL = f'Version {gdal.__version__}'
-
-        # create dimensions
-        crs = parse_crs_from_ogc_wkt(ds_in.GetProjectionRef())
-        if crs.cs_type == 'Projected':
-            _process_projected(ds_in, dst)
-        else:
-            _process_geogcs(ds_in, dst)
+        with OpenGDAL(infile) as input_geotiff, \
+            Dataset(outfile, mode='w', format='NETCDF4') as output_netcdf4:
+            # define global attributes
+            output_netcdf4.title = ''
+            output_netcdf4.institution = 'Alaska Satellite Facility'
```

Comment:
And do we still want ASF in here?

Reply:
Good point - I'll do some digging in some of our on-prem services to see what we put there (if we put anything).